README.md

```diff
@@ -1,2 +1,141 @@
+Multiversx blockchain.
+
+To simplify the process of setting up your own data receiver from the Multiversx blockchain node, we have created this
+repository as a comprehensive template. It serves as a valuable resource, guiding you through the necessary steps to
+establish a successful connection with the Multiversx blockchain node and receive the exported data effortlessly.
+
+Whether you are building a decentralized application, monitoring the blockchain network, or conducting in-depth data
+analysis, this template repository equips you with the foundation to swiftly integrate and leverage the exported data
+from the Multiversx blockchain node.
+
+Begin your journey towards seamless data integration and take advantage of the Outport Driver's robust websocket
+connector by exploring the contents of this repository. We look forward to witnessing the innovative applications and
+services that will emerge as a result of this collaborative ecosystem.
+
+_Note: The Outport Driver is an open-source project, and we encourage active contributions and feedback from the
+community to further enhance its capabilities and compatibility with different blockchain networks._
+
+## How to use
+
+### Use cases
+
+The Outport Driver operates on a robust websocket connection architecture, supporting both server and client roles. The
+system is designed to seamlessly export data from the Multiversx blockchain node, which adopts a sharded architecture
+where each shard represents a separate running chain.
+
+To export data from a specific shard, you need to enable an observer node within that shard to export data. One can do
+that
+by enabling the **[HostDriverConfig].Enabled** flag to true
+[from this config file](https://github.com/multiversx/mx-chain-go/blob/master/cmd/node/config/external.toml)
+
+In our case of a network consisting of three shards, you would require three separate websocket receiver binaries, each
+responsible for receiving data from its respective shard. This distributed approach ensures data integrity and allows
+for shard-specific data processing.
+
+However, the websocket connection in the Outport Driver is highly flexible and parameterizable. By customizing the
+configuration, you can configure the observer nodes to function as clients and the driver (this template binary) as a
+server. This configuration reduces the number of driver receiver binaries to just one, acting as a server that receives
+data from each observer, from each shard. To set your receiver as a server, you need set **mode**
+from `cmd/connector/config/config.toml` to **"server"**.
+
+Alternatively, you have the option to set up the observer node as a server capable of handling multiple clients. This
+setup proves advantageous when multiple services within your ecosystem need to receive exported data. For instance, a
+single node could export data to various services, such as an elastic indexer, a monitoring tool, a notifier, and more.
```

Comment:
`, and more` -> `or other implementations`

Reply:
Right, changed as suggested